oauth_provider/runtests/manage.py

```diff
@@ -1,16 +1,28 @@
 #!/usr/bin/env python
 # put django-oauth in PYTHONPATH
-import sys, os
-sys.path = [os.path.join(os.getcwd(), '../../')] + sys.path
+from __future__ import absolute_import
 
-from django.core.management import execute_manager
+import os
+import sys
 
-try:
-    import settings # Assumed to be in the same directory.
-except ImportError:
-    import sys
-    sys.stderr.write("Error: Can't find the file 'settings.py' in the directory containing %r. It appears you've customized things.\nYou'll have to run django-admin.py, passing it your settings module.\n(If the file settings.py does indeed exist, it's causing an ImportError somehow.)\n" % __file__)
-    sys.exit(1)
+# from django.core.management import execute_manager
```

Comment:
@mraarif why you comment out all this ?

Reply:
this should be removed, deprecated way of how django's `manage.py` used to work, it caused runtime errors on newer versions of django